### DISCUSSION TITLE
Duda sobre POST

### DISCUSSION THREAD

**Discussion by nubevi** [dds-jv/foro#264] [TPA - 2021] [answered]
Buenas noches,

Me encontraba codeando el POST para el tp y me dí cuenta que hay un response medio extraño (no se me ocurre como modelarlo, al menos de que sea un error...)

![image](https://user-images.githubusercontent.com/26943847/120132170-3bf87980-c1a0-11eb-8302-bf931c665664.png)

Desde ya,

Muchas gracias por la ayuda :)

**Comment by roli93** [comment, marked as answer]
Efectivamente y como veremos en la 2da mitad de la materia, los códigos de respuesta iniciadoa con 4xx, representan errores. Fijate que en ese caso lo interesante el el codigo de respuesta (409, Conflict) que ya tiene toda la informacion necesaria sobre el error, por eso el body tiene básicamente cualquier saraza

**Reply by gastonprieto** [reply]
te sirvio @nubevi ?

**Reply by nubevi** [reply]
Si, colgué!
